tools/myst/host/exec.c

```diff
@@ -455,13 +460,16 @@ int exec_action(int argc, const char* argv[], const char* envp[])
             cli_getopt(&argc, argv, "--strace", NULL) == 0)
         {
             options.strace_config.trace_syscalls = true;
+            // fprintf(stderr, "Failed to  final options\n");
```

Comment:
Should we remove commented lines?

Reply:
this isnt a part of the final PR, not sure how you see it here